tasks/install-plugins.yml

```diff
@@ -27,26 +27,45 @@
       # TODO: Remove this suppression when ansible-lint learns to parse block conditions
       # See: https://github.com/ansible/ansible-lint/issues/1894
       # noqa no-changed-when
-      ansible.builtin.command: >
+      ansible.builtin.shell: >
         java -jar
         {{ jenkins_pimt_path }}
         --plugin-download-directory {{ jenkins_plugins_dir }}
         --plugin-file {{ jenkins_config_dir }}/plugins.yaml
         --war {{ jenkins_war }}
         --verbose
         {{ jenkins_pimt_args }}
+        {{ (jenkins_pimt_logfile != "") | ternary("2> /tmp/pimt.log", "") }}
       environment:
         CACHE_DIR: "{{ jenkins_caches_dir }}"
+      # Weirdly, no_log wants to interpret `jenkins_pimt_logfile != ''` as a string, not
+      # a boolean. So we need to use the ternary filter again.
+      no_log: "{{ (jenkins_pimt_logfile != '') | ternary(true, false) }}"
       retries: "{{ jenkins_download_retries }}"
       delay: "{{ jenkins_download_delay }}"
       register: plugin_installation_result
       until: plugin_installation_result is succeeded
+
+    # The `fetch` module doesn't let you write files to arbitrary names, so we need to use
+    # `slurp`/`copy` to write the data ourselves.
+    - name: Read PIMT logfile to Ansible control machine
+      ansible.builtin.slurp:
+        src: "/tmp/pimt.log"
+      when: jenkins_pimt_logfile != ""
+      register: jenkins_pimt_logfile_contents
```

Comment:
If we do the `register` on the previous task, we don't need to pipe to `/tmp/pimt.log`, and we can keep on using `ansible.builtin.command`

Reply:
Yeah, but then the output still floods the Ansible logs, which is one of the issues this PR is meant to address.

Reply:
Not if you have `no_log`. My point is to have both `no_log` and `register`.

Reply:
Ah, that's pretty clever! I didn't think that this would work... I assumed that `no_log` would also remove the `stdout` and `stderr` from the task results, but it doesn't. Fixed in fdc98c2.